pynamodb/connection/base.py

```diff
@@ -109,6 +109,24 @@ def get_index_hash_keyname(self, index_name):
                     if schema_key.get(KEY_TYPE) == HASH:
                         return schema_key.get(ATTR_NAME)
 
+    def get_index_range_keyname(self, index_name):
+        """
+        Returns the name of the hash key for a given index
+        """
+        global_indexes = self.data.get(GLOBAL_SECONDARY_INDEXES)
+        local_indexes = self.data.get(LOCAL_SECONDARY_INDEXES)
+        indexes = []
```

Comment:
do we do this index aggregation anywhere else? if so, could be good to abstract to a func / lazy property